corehq/apps/app_manager/static/app_manager/js/xpathConfig.js

```diff
@@ -0,0 +1 @@
+../../../xpath_validator/nodejs/xpathConfig.js
```

Comment:
This part I'm iffy about. Will collect static follow a symlink or copy the symlink (or neither)? This'll only work if it follows